apps/web/cypress/tests/templates-store.spec.ts

```diff
@@ -12,7 +12,6 @@ describe('Templates Store', function () {
   };
 
   beforeEach(function () {
-    cy.clearDatabase();
```

Comment:
My experience is this was showing up some false positives. I can still replicate the same failures in the local environment even with its removal.